.github/workflows/main.yml

```diff
@@ -27,8 +27,8 @@ jobs:
           java-version: 17
           distribution: temurin
       - name: Release Maven package
-        uses: samuelmeuli/action-maven-publish@v1
-        with:
-          nexus_username: ${{ secrets.SONATYPE_OSSRH_USERNAME }}
-          nexus_password: ${{ secrets.SONATYPE_OSSRH_PASSWORD }}
-          maven_profiles: "branch"
\ No newline at end of file
+        #run: ./gradlew publishJvmPublicationToSonatypeRepository
+        run: ./gradlew assemble
```

Comment:
Publication is commented out

Reply:
Any particular reason to not publish?

Reply:
Not really, the only reason was that I wasn't sure you want to release it as `1.1.0-SNAPSHOT` or something else. The PR will automatically release the SNAPSHOT version, that's why I commented it out.

Reply:
We can remove the comment once we are ready for first fully converted version.